app/modules/onboarding/assets/js/pages/account.js

```diff
@@ -115,6 +115,13 @@ export default function Account() {
 		navigate( 'onboarding/' + nextStep );
 	};
 
+	function getNextStep() {
```

Comment:
```suggestion
	const getNextStep = () => {
```
Not critical. You can ignore if you want.

Reply:
`const`s are not hoisted in the `global` scope.
So if this function would be a const it wouldn't be accessible as I use this function before it is declared.